interactions/client.py

```diff
@@ -183,10 +183,10 @@ async def __bulk_update_sync(self, data: List[dict], delete: Optional[bool] = Fa
 
         for command in data:
             if command.get("guild_id"):
-                if guild_commands.get(command["guild_id"]):
-                    guild_commands[command["guild_id"]].append(command)
+                if guild_commands.get(int(command["guild_id"])):
+                    guild_commands[int(command["guild_id"])].append(command)
                 else:
-                    guild_commands[command["guild_id"]] = [command]
+                    guild_commands[int(command["guild_id"])] = [command]
```

Comment:
No

Reply:
Agreed

Reply:
other than that there's nothing to change? I di d that because the input sometimes the command["guild_id"] will be string so to make it easier I did it to int

Reply:
It's gonna be a string all the time, snowflakes are transported as strings over the api, no need to waste processing over something internal

Reply:
but when i tested if some are string some are int the synchro will not be done and some will not be loaded